cylc/flow/scripts/broadcast.py

```diff
@@ -312,12 +317,33 @@ def get_option_parser() -> COP:
         help="Use unicode box characters with -d, -k.",
         action="store_true", default=False, dest="unicode")
 
+    # BACK COMPAT: --raw
+    # From: < 8.5.1
+    # To: 8.5.1
+    # Remove at: 8.7.0
     parser.add_option(
         "-r", "--raw",
-        help="With -d/--display or -k/--display-task, write out "
-             "the broadcast config structure in raw Python form.",
+        help=(
+            "With -d/--display or -k/--display-task, write out "
+            "the broadcast config structure in raw Python form. "
+            f"{RAW_DEPR_MSG}"
+        ),
         action="store_true", default=False, dest="raw")
 
+    parser.add_option(
+        '--format',
```

Comment:
Consider adding the short option `-t`.

Reply:
I had a look at scripts that we might implement `--format` for:
- `broadcast`: No conflict for `-t` in this case. However...
- `dump`: `-t` is `--tasks`
- `list`: `-t` is `--tree`
- `play`: (which already has `--format`) `-t` is `--start-task`
- `graph`: `-t` is `--transpose`
- `workflow-state`: `-t` is `--task`

So I'm not sure there is going to be consistency with `scan`